array-map/main.js

```diff
@@ -0,0 +1,14 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+const price = numbers.map(x => `$${x}.00`);
+const upperCased = languages.map(x => x.toUpperCase());
+const firstLetters = languages.map(x => x.charAt(0));
```

Comment:
A better parameter name than `x` would be the name of the array in singular form (`number` or `language`)